landing-page/src/App.js

```diff
@@ -0,0 +1,61 @@
+import './App.css';
```

Comment:
material-ui has support for integrating css in the Js file: https://material-ui.com/styles/basics/